### PR TITLE
Dco data

### PR DESCRIPTION
**Pull request by mrpatrickwest**
dataset, distribution and data repository

### CODE REVIEW

**Comment by mrpatrickwest** [comment]
still a few problems that need to be worked out. And still not sure how to represent the creator and contributor properties so that they are easier to enter in VIVO.

**Comment by mrpatrickwest** [comment]
problems
1. wasQuotedFrom not seeing documents, even though I added a restriction for bibo:Document
2. dct:temporal is not allowing me to create a new period of time, not showing up as an option
3. dct:accrualPeriodicity is not allowing me to create a new frequency, not showing up as an option

**Comment by mrpatrickwest** [comment]
wasQuotedFrom, landingPage, accesURL and downloadURL are currently the only problems remaining. It's in the ontology, just need code changes in VIVO to get it to work. Same with the creator and collaborator properties. Changes need to be made in VIVO code to get this to work properly.

**Comment by mrpatrickwest** [comment]
@zednis @olyerickson @xgmachina @am-e @zhonghao1014 Please review the ttl files. I've gotten it all loaded into test and migrated the current datasets to the new schema. Still coding to do, but that'll be in dco-vivo branch dco-data.

It all looks good, the ttl files all validate and load. All the classes and properties are there.

Next steps are to 1) get the properties showing up properly, 2) custom forms for the simpler properties like accessURL and downloadURL and landingPage, 3) custom form for dataset and distribution using one form, 4) custom form for distribution using one form, 5) update elasticsearch for datasets, 6) update drupal page queries and xslt to display properly.

Once we get it all working in test then we can push it all to production. Can't push the ontology changes then the VIVO changes then the elasticsearch changes then the drupal page changes.

File: dco.ttl [hunk 1]
```diff
@@ -1630,3 +1640,23 @@ dco:cites
       The entity has a specific reference, such as a quotation or dataset, from another entity
       """^^rdf:XMLLiteral .
 
+dco:Contributor
+      a       owl:Class ;
+      rdfs:label "Contributor"@en-US ;
+      rdfs:subClassOf core:Relationship ;
+      rdfs:subClassOf
+              [ a       owl:Restriction ;
+                owl:allValuesFrom foaf:Agent ;
```
Comment (zednis) [inline dco.ttl:1649]: should this be `owl:allValuesFrom`?

**Pull request merged**